Tools-Override/tests.targets

```diff
@@ -137,10 +137,12 @@
       <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
     </GetTargetMachineInfo>
     <PropertyGroup>
-      <TestDisabled Condition="'%(UnsupportedPlatformsItems.Identity)' == '$(TargetOS)'">true</TestDisabled>
+      <TestDisabled Condition="'%(UnsupportedPlatformsItems.Identity)' == '$(TargetOS)' Or '$(ConfigurationErrorMsg)' != ''">true</TestDisabled>
     </PropertyGroup>
     <Message Condition="'%(UnsupportedPlatformsItems.Identity)' == '$(TargetOS)'"
       Text="Skipping tests in $(AssemblyName) because it is not supported on $(TargetOS)" />
+    <Message Condition="'$(ConfigurationErrorMsg)' != ''"
+      Text="Skipping tests in $(AssemblyName) because there is no configuration compatible with the current BuildConfiguration." />
```

Comment:
Why are we using ConfigurtionErrorMsg to determine this? Wouldn't it be better to use the absence of the Configuration?